js/sliderView.js

```diff
@@ -180,107 +182,107 @@ define([
         return;
       }
 
-      var itemValue = parseFloat($(event.currentTarget).attr('data-id'));
-      var index = this.getIndexFromValue(itemValue);
+      const itemValue = parseFloat($(event.currentTarget).attr('data-id'));
+      const index = this.getIndexFromValue(itemValue);
       this.selectItem(index);
       this.animateToPosition(this.mapIndexToPixels(index));
       this.setSliderValue(itemValue);
-    },
+    }
 
-    getValueFromIndex: function(index) {
+    getValueFromIndex(index) {
       return this.model.get('_items')[index].value;
-    },
+    }
 
-    resetControlStyles: function() {
+    resetControlStyles() {
       this.$('.slider-handle').empty();
       this.showScaleMarker(false);
-      this.$('.slider-bar').animate({width:'0px'});
+      this.$('.slider-bar').animate({ width: '0px' });
       this.setSliderValue(this.model.get('_items')[0].value);
-    },
+    }
 
-    onCannotSubmit: function() {},
+    onCannotSubmit() { }
 
-    setSliderValue: function (value) {
+    setSliderValue(value) {
```

Comment:
This function is an example of an easy "return early".
The main body of this function is:
```js
this.$slider.val(value).change();
```

The main body should only be executed if the given condition is met:
```js
if (this.$slider) { }
```
If we therefore invert the condition and return early, we can reduce the nesting and reduce number of lines whilst increasing the clarity:
```js
if (!this.$slider) return;
this.$slider.val(value).change();
```